index.html

```diff
@@ -0,0 +1,183 @@
+      <section class="hero" id="hero-content">
+        <div class="hero-inner-content inner-content">
+          <h2>"Hello, Tech fans"</h2>
+          <div class="greeting">
+            <h1 class="large-text">
+              Dublin <br /> Tech Summit <br />
+              <span>2022</span>
+            </h1>
+          </div>
+          <div class="hero-inner-text">
+            <p>
+              Dublin Tech Summit (DTS) is a two-day conference that sits at the heart of the international tech scene. As one of the world’s most active technology hubs, Dublin has become the EMEA base for some of the biggest global tech companies and over the last 5 years DTS has emerged as one of Europe’s fastest growing B2B tech events.
+            </p>
+          </div>
+          <div class="venue-details">
+            <p class="summit-date">2022.06.15(Wed) ~ 16(Thu)</p>
+            <p class="summit-venue">
+              @ Ballsbridge, Royal Dublin Society.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="program" id="main-program">
+        <div class="section-header">
+          <h2>main program</h2>
+          <hr class="orange-line" />
+        </div>
+        <div class="program-wrapper inner-content">
+          <div class="program-tile">
+            <i class="fa-solid fa-chalkboard-user"></i>
+            <h3>lecture</h3>
+            <p>
+              Listen to the speakers from various countries about the messages
+              of sharing and opening
+            </p>
+          </div>
+          <div class="program-tile">
+            <i class="fa-solid fa-chalkboard"></i>
+            <h3>f1 exhibition</h3>
+            <p>
+              Appreciate various drivers applying for their F1 driver's
+              licences, organized by Silverston circuit
+            </p>
+          </div>
+          <div class="program-tile">
+            <i class="fa-solid fa-message"></i>
+            <h3>Forum</h3>
+            <p>
+              Have time for attendees to share their thoughts and opinions with
+              F1 drivers, and the conveference committee
+            </p>
+          </div>
+          <div class="program-tile">
+            <i class="fa-solid fa-users"></i>
+            <h3>Workshop</h3>
+            <p>
+              We will deliver another world-class conference and will adhere to all government Covid safety guidelines.
+            </p>
+          </div>
+          <div class="program-tile">
+            <i class="fa-solid fa-spinner"></i>
+            <h3>f1 ignite</h3>
+            <p>
+              Get opportunities to network with F1 affiliates around the world,
+              also after the summit.
+            </p>
+          </div>
+        </div>
+        <a href="#" class="join-button">Join DTS Community of Leaders</a>
+        <a href="#" class="view-more">see the whole program</a>
+      </section>
+      <section class="featured-speakers" id="speakers"></section>
```

Comment:
- [ ] In this featured speakers section your text seems unreasonably smaller. please increase the `font-size` of this text as it will make it a lot easier to read. do this for each featured speaker
- [ ] Make this change anywhere else where your text is too small. text `font-size` of `10px` in sections of  important text is too small.

![image](https://user-images.githubusercontent.com/18730106/158866493-0adc9a55-5ac9-4aa3-954e-8150c7ca96ac.png)

Reply:
All good now 👍🏽